samples/bluetooth/beacon_dp/README.rst

```diff
@@ -0,0 +1,27 @@
+.. _bluetooth-beacon-dp-sample:
+
+Bluetooth: Beacon Dynamic Tx Power
+##################################
+
+Overview
+********
+
+A simple application demonstrating the dynamic Tx power control of the LL BLE controller over a BLE Broadcaster node by
```

Comment:
Let's make this a sentence:
```
This sample application demonstrates the dynamic ...
```

Reply:
Rewrote the sample docs given the changes provided over the HCI commands. Now available under 3bf7dda34231e3991dfd927e4c8a1477163ce797